src/k8s-engine/wsl.ts

```diff
@@ -1338,9 +1338,10 @@ export default class WSLBackend extends events.EventEmitter implements K8s.Kuber
             Any files named '*.start' in this directory will be executed
             sequentially on Rancher Desktop startup, before the main services.
             Files are processed in lexical order, and startup will be delayed
-            until they have all run to completion. Similaryly, any files named
+            until they have all run to completion. Similarly, any files named
             '*.stop' will be executed on shutdown, after the main services have
             exited, and delay shutdown until they have run to completion.
+            Note that the script file names may not include whitespace.
```

Comment:
Versioning problem at line 1335: if the contents of `provisioning/README` gets changed in the future, this code won't update it.